docs/UserGuide.md

```diff
@@ -124,11 +124,11 @@ Step 5 : Start using the app!
 <a name="parameter"></a>
 ## Common Parameters
 
-| Parameters | Description        | Constraints                                                                                                                                                       | Valid Examples                  | Invalid Examples             |
-|------------|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------|------------------------------|
-| `n/`       | Specifies the name | Must be alphanumeric and can contain spaces                                    <br/>                                                                              | John Doe                        | $5money                      |
-| `a/`       | Telegram handle    | Must be a-z, 0-9 or underscore. <br/> Minimum Length is 5 characters and maximum 32 characters <br/> Cannot end with an underscore <br/> Must start with a letter | harukaNaruto, loli_pop, l0l1pop | $telegram, _tele, 56thperson |
-
+| Parameters | Description     | Constraints                                                                                                                                                       | Valid Examples                 | Invalid Examples
+|------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------| -------------- 
+| `n/`       | Student names   | Must be alphanumeric and can contain spaces                                    <br/><br/>                                                                         | John Doe                       | $5money        
```

Comment:
`n/` doesn't seem to only be used for student names. It is a pretty general notion of a name, which can also be used for assignments for example.